crates/dbs-miniball/src/vmm/src/boot.rs

```diff
@@ -192,76 +236,19 @@ mod tests {
             (GuestAddress(MMIO_GAP_END), 0x1000),
         ])
         .unwrap();
+        let address_space = create_address_space(GuestAddress(MMIO_GAP_START), 1024).unwrap();
         assert_eq!(
             build_bootparams(
                 &guest_memory,
-                &kern_load_res,
+                &address_space,
                 GuestAddress(MMIO_GAP_START + 1),
                 GuestAddress(MMIO_GAP_START),
-                GuestAddress(MMIO_GAP_END)
+                GuestAddress(MMIO_GAP_END),
+                GuestAddress(cmdline_addr),
+                (kernel_cfg.cmdline.as_str().len() + 1) as usize,
             )
             .err(),
             Some(Error::HimemStartPastMmioGapStart)
         );
-
-        // Success case: 2 ranges surrounding the MMIO gap.
-        // Setup header is specified in the kernel loader result.
-        kern_load_res.setup_header = Some(bootparam::setup_header::default());
```

Comment:
Why does these tests removed?

Reply:
Because I didn't use kernel_result to get hdr, I removed the corresponding UT. However, the successful case was not retained when it was removed, and it is now added.